packages/library/src/commonMain/kotlin/io/realm/internal/RealmObjectUtil.kt

```diff
@@ -101,14 +102,10 @@ internal fun <T : RealmObject> RealmObjectInternal.thaw(liveRealm: BaseRealm): T
                 thawedObject
             )
         }
-    } catch (e: Exception) {
+    } catch (exception: RealmCoreUnknownException) {
```

Comment:
Is this right? I would expect error code 2 to be mapped to RealmCoreOtherException 🤔

Reply:
It is addressed in #399